src/App.js

```diff
@@ -1,26 +1,75 @@
 import React from 'react';
-import logo from './logo.svg';
+//import logo from './logo.svg';
+import 'bulma/css/bulma.css';
 import './App.css';
+import foods from './foods.json';
+import FoodBox from './components/FoodBox';
+import AddFood from './components/AddFood';
+import TodaysFood from './components/TodaysFood'
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+
+class App extends React.Component {
+
+  state = {
+    foodList: foods,
+    search: false
+  }
+  handleAddFood = (event) =>{
+    event.preventDefault();
+    let inputs = event.currentTarget
+    this.setState({
+      foodList: [{name: inputs.name.value, calories: inputs.calories.value, image: inputs.image.value, quantity:0}, ...this.state.foodList],
```

Comment:
suggestion for readability: create the object in a line above and then insert it in the state.